README.md

```diff
@@ -13,6 +13,7 @@ Currently, GrimoireLab is organized in the following repositories:
 * [Perceval (bundle for OPNFV)](https://github.com/chaoss/grimoirelab-perceval-opnfv)
 * [Perceval (bundle for Mozilla)](https://github.com/chaoss/grimoirelab-perceval-mozilla)
 * [Perceval (bundle for Puppet)](https://github.com/chaoss/grimoirelab-perceval-puppet)
+* [Graal](https://github.com/chaoss/grimoirelab-graal): retrieval of data from source code
```

Comment:
What about "source data analysis with external tools"?

Reply:
sorry for the late reply, it seems that some notifications are forwared to my email with 1 day of delay

Reply:
No problem. What do you think about my suggestion for that slight change in the text?

Reply:
Yes, sure. I have just updated the PR.